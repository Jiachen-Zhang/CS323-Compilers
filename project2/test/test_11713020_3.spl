int a()
{
  return 0;
}
int test_11713020_3()
{
  int a;
  return 0;
}