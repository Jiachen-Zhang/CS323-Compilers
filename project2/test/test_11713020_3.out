Error type 3 at Line 7: redefine variable: a
